src/main/java/seedu/address/logic/Messages.java

```diff
@@ -26,9 +26,9 @@ public class Messages {
                 "Oops! Multiple values are specified for the following single-valued parameter(s): ";
 
     public static final String MESSAGE_NON_INTEGER_INDEX = "Oops! The index you provided is not valid.\n"
-            + "It appears to be either not a valid integer or it exceeds the maximum allowed value "
-            + "(2147483647).\n"
-            + "Please try again with a valid positive integer!";
+            + "It should be a positive integer and within the range of 1 to "
+            + Integer.MAX_VALUE + ".\n"
+            + "Please try again with a valid index!";
```

Comment:
Should we state what is Integer.MAX_VALUE?

Reply:
That is just how it shows in the code, the actual value is displayed to the user